packages/khalti/test/khalti_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:khalti/khalti.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:test/test.dart';
+
+class _MockKhaltiService extends Mock implements KhaltiService {}
+
+void main() {
+  const _testPublicKey = '__test_public_key__';
+  final _config = KhaltiConfig(
```

Comment:
Since these variables are already instead a method, there is no way it can be accessed outside from the method.
So consider removing `_`.

Reply:
Silly me. 😂